src/Annotator.tsx

```diff
@@ -938,10 +939,16 @@ export class Annotator extends React.Component<Props, State>{
         return data;
     };
 
+    onReload = ()=> {
```

Comment:
It's better to write this reloading logic in the parent component, where you can switch to the next image without refreshing the page...